### ISSUE TITLE
Travis is all messed up

### ISSUE THREAD

**Issue by adamkaplan** [yahoo/YMPromptKit#3]

**Comment by dbgrandi** [comment]
Not sure if I'd consider this fixed. From the log output, I don't think it actually builds the code.

https://travis-ci.org/yahoo/YMPromptKit

**Comment by dcaunt** [comment]
Indeed. For now it at least stops try to build as ruby. We need a basic script or some config to tell Travis which workspace & scheme to build.

**Comment by dbgrandi** [comment]
Here's what I use:

https://github.com/dbgrandi/DBGHTMLEntities/blob/master/.travis.yml

**Comment by dcaunt** [comment]
Thanks @dbgrandi that was useful. It's working now.
